core/src/rpc.rs

```diff
@@ -1524,21 +1528,24 @@ impl RpcSol for RpcSolImpl {
 }
 
 fn deserialize_bs58_transaction(bs58_transaction: String) -> Result<(Vec<u8>, Transaction)> {
-    let wire_transaction = bs58::decode(bs58_transaction).into_vec().unwrap();
```

Comment:
ffs, I was literally in this code the other day and looked right past this unwrap

Reply:
Yeah, same.
I just audited all the other unwraps in rpc.rs